src/homedelivery/query.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+import { fetchConfig } from './../lib/config'
+import type {Config} from './../lib/config'
+import {Zuora} from './../lib/Zuora'
+import type {Query} from './../lib/Zuora'
+import moment from 'moment'
+import type { input } from '../querier'
+
+async function queryZuora (deliveryDate, config: Config) {
+  const formattedDate = deliveryDate.format('YYYY-MM-DD')
+  const deliveryDay = deliveryDate.format('dddd')
+  const zuora = new Zuora(config)
+  const subsQuery: Query =
+    {
+      'name': 'Subscriptions',
+      'query': `
+        SELECT
+            RateplanCharge.quantity,
+            Subscription.Name,
+              SoldToContact.Address1,
+              SoldToContact.Address2,
+              SoldToContact.City, 
+              SoldToContact.Country, 
+              SoldToContact.FirstName, 
+              SoldToContact.LastName, 
+              SoldToContact.PostalCode, 
+              SoldToContact.State,
+              SoldToContact.workPhone,
+              Account.SpecialDeliveryInstructions__c
+          FROM
+            rateplancharge
+          WHERE
+           (Subscription.Status = 'Active' OR Subscription.Status = 'Cancelled') AND
+           ProductRatePlanCharge.ProductType__c = 'Print ${deliveryDay}' AND
+           Product.Name = 'Newspaper Delivery' AND
+           RatePlanCharge.EffectiveStartDate <= '${formattedDate}' AND
+           RatePlanCharge.EffectiveEndDate >= '${formattedDate}' AND
```

Comment:
Add a comment here to say that the = deliberately catches the case where we want to fulfil on the actual renewal date, but Zuora won't renew the sub till part way through the day.